picopass/files/iclass_elite_dict.txt

```diff
@@ -1,6 +1,10 @@
 
 ## From https://github.com/RfidResearchGroup/proxmark3/blob/master/client/dictionaries/iclass_default_keys.dic
 
+#FORMAT GUIDE: Never add # after a key.
```

Comment:
Are you addressing this message to developers or users?

Asking because those files are not user editable and will be overwritten with app update.

Reply:
Both developers and users.
These files are user editable afaik as they allow the users to add their own custom keys.

Reply:
Users _should_ all their own keys to `iclass_elite_dict_user.txt`, but some people may just see these and overwrite them, as @Antiklesys literally did to discover the commenting issue.

Reply:
Shouldn't we mention it and provide example files how it's done in subghz?

Reply:
I think the best way was if the flipper was actually able to process correctly inline comments and spaces following the keys but I guess this is the second best thing